shopservice/src/main/java/moviecatalog/shopservice/controller/ShopController.java

```diff
@@ -64,4 +64,10 @@ public int getTotalCountOfBoughtMoviesForUser(final @PathVariable Long userId) {
         LOGGER.info("get total movies bought by user with id '{}'", userId);
         return movieOrderService.getTotalOrdersCountForUser(userId);
     }
+
+    @GetMapping("/movie/count/{movieId}")
```

Comment:
Možná bych ten endpoint pojmenoval trošku jinak. Myslel jsem, že ti to počítá filmy a ne prodané filmy.

Reply:
diky za tip :)